Add copying of application code back into deploy dockerfile

dancoates opened the pull request:
This was accidentally removed in PR #957, causing the container to not start correctly due to missing application code

jmarshall commented:
Is it that this was inadvertently / incorrectly removed in PR #957?

(It'd be great to have that context — and what went wrong, if anything — in the commit message or as a comment here.)

Also they were previously in alphabetical order, so it'd help diffing a tiny bit if this put them back identically.

dancoates commented:
Thanks @jmarshall good points, I've added a reference to the PR in the commit, and fixed the ordering to match what it was previously for better diff/blame.

jmarshall reviewed (approved):
Perfect, thanks!
